torch_frame/nn/encoder/stype_encoder.py

```diff
@@ -15,6 +22,14 @@
 
 from ..utils.init import attenuated_kaiming_uniform_
 
+NUM_DAYS_PER_MONTH = 31
+NUM_MONTHS_PER_YEAR = 12
+NUM_DAYS_PER_MONTH = 31
```

Comment:
Duplicated? Also some months don't have 30 days right?

Reply:
ohh good catch. Yeah but I want to keep the scale between 0 and 1, so I divide by the largest possible number of days

Reply:
Might be good to add this as a comment, potentially with a TODO to revisit this.